src/main/java/zosjobs/CancelJobs.java

```diff
@@ -0,0 +1,70 @@
+package zosjobs;
+
+import core.ZOSConnection;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import rest.*;
+import utility.Util;
+import utility.UtilIO;
+import zosjobs.input.CancelJobParams;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class CancelJobs {
+
+    private static final Logger LOG = LogManager.getLogger(CancelJobs.class);
+    private final ZOSConnection connection;
+
+    /**
+     * CancelJobs Constructor
+     *
+     * @param connection connection information, see ZOSConnection object
+     * @author Nikunj Goyal
+     */
+    public CancelJobs(ZOSConnection connection) {
+        this.connection = connection;
+    }
+
+    /**
+     * Cancel a job that resides in a z/OS data set.
+     *
+     * @return job document with details about the submitted job
+     * @throws Exception error on submitting
+     * @author Frank Giordano
+     */
+    public Response cancelJob() throws Exception {
+        return this.cancelJobsCommon(null);
```

Comment:
If params are null you will have an exception here:
```
Util.checkNullParameter(parms == null, "parms is null");
```
just create an empty params object here